docs/src/main/sphinx/installation/cli.rst

```diff
@@ -358,8 +360,76 @@ The following table list the available options for Kerberos authentication:
   * - ``--krb5-service-principal-pattern``
     - Remote kerberos service principal pattern (default: ${SERVICE}@${HOST})
 
-See :doc:`/security/cli` for more information on configuring and using Kerberos
-with the CLI.
+Kerberos principals and keytab files
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Each user, who connects to the Trino coordinator, needs a Kerberos principal.
+You need to create these users in Kerberos using `kadmin
+<http://web.mit.edu/kerberos/krb5-latest/doc/admin/admin_commands/kadmin_local.html>`_.
+
+Additionally, each user needs a `keytab file
+<http://web.mit.edu/kerberos/krb5-devel/doc/basic/keytab_def.html>`_. The
+keytab file can be created using :command:`kadmin` after you create the
+principal.
+
+.. code-block:: text
+
+    kadmin
+    > addprinc -randkey someuser@EXAMPLE.COM
+    > ktadd -k /home/someuser/someuser.keytab someuser@EXAMPLE.COM
+
+.. include:: ../security/ktadd-note.fragment
+
+Configuration for TLS
+~~~~~~~~~~~~~~~~~~~~~
+
+When using Kerberos authentication, access to the Trino coordinator must be
+through :doc:`HTTPS and TLS </security/tls>`.
+
+Troubleshooting
+~~~~~~~~~~~~~~~
+
+Many of the same steps, that can be used when troubleshooting the :ref:`Trino
+coordinator <coordinator-troubleshooting>` apply to troubleshooting the CLI.
+
+Additional Kerberos debugging information
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+You can enable additional Kerberos debugging information for the Trino CLI
+process by passing ``-Dsun.security.krb5.debug=true``,
+``-Dtrino.client.debugKerberos=true``, and
+``-Djava.security.debug=gssloginconfig,configfile,configparser,logincontext``
+as a JVM argument, when starting the CLI process. Doing so requires invoking
```

Comment:
Again, where do these random commas come from? Use a comma where you would take a breath when reading aloud. This one goes out.

Reply:
lol @Ordinant I don't know. Hasty copying and pasting and chopping, maybe. I'll slap a label on this PR, edit, and check links and such.